apps/kserve/charts/kserve-0.14.1/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: kserve
+description: A Helm chart to refer the official "kserve" helm chart
+type: application
+version: v0.14.1
+dependencies:
+  - name: kserve
+    version: v0.14.1
+    repository: oci://ghcr.io/kserve/charts 
```

Comment:
Please fix "No newline at end of file" issue in all added files.

Reply:
Done.